_docs/setup/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Setup
+overview: Installation instructions for Istio control plane and sidecar on different platforms.
```

Comment:
I like the overview you have on the Welcome/index: "Setup contains instructions for installing the Istio control plane in various environments (e.g., Kubernetes, Consul, etc.), as well as instructions for installing the sidecar in the application deployment." Making them the same would also increase consistency.